pkg/reconcilers/deploymentconfig.go

```diff
@@ -246,3 +247,17 @@ func DeploymentConfigPriorityClassMutator(desired, existing *appsv1.DeploymentCo
 
 	return updated, nil
 }
+
+// DeploymentConfigTopologySpreadConstraintsMutator ensures TopologySpreadConstraints is reconciled
+func DeploymentConfigTopologySpreadConstraintsMutator(desired, existing *appsv1.DeploymentConfig) (bool, error) {
```

Comment:
Can we test this? It should not be hard

Reply:
Done (new commit). Thank you for comment.